src/python/pants/backend/python/goals/export.py

```diff
@@ -191,7 +191,10 @@ async def do_export(
                         "--remove=all",
                         output_path,
                     ],
-                    {"PEX_MODULE": "pex.tools"},
+                    {
+                        **complete_pex_env.environment_dict(python_configured=True),
```

Comment:
It probably makes sense to also use `complete_pex_env.create_argv` above. In that case there is no bug fix afaict, but you also open yourself up to DRY bugs going forward by not using it.

Reply:
Good idea, done.